board/sunxi/cubie_board.c

```diff
@@ -0,0 +1,49 @@
+#include <asm-generic/gpio.h>
+#include <sunxi_gpadc.h>
+#include <dm/device.h>
+
+#define countof(x) (sizeof(x) / sizeof(x[0]))
+#define HW_ID_ADC_CHANNEL	2
+/*PK 24 */
+#define HW_ID_GPIO			344
+
+struct hw_info_def {
+	char *compatible;
+	char *fdtfile;
+	unsigned int hw_id_upper_bound;
+	unsigned int hw_id_lower_bound;
+	unsigned int hw_id_level;
```

Comment:
```suggestion
	unsigned int hw_id_level;
	unsigned int hw_id_lower_bound;
	unsigned int hw_id_upper_bound;
```
一般是下界在前上界在后，然后gpio我们相当于是最高有效位，这样后面产品列表排列起来顺一点